datasources/includes/VIC.ejs

```diff
@@ -323,9 +323,13 @@
       "minimumMaxScaleDenominator": 10000000000,
       "filterByWmsGetCapabilities": true,
       "filterQuery": [
-        "fq=res_format%3awms"
+        "fq=+(res_format%3Awms%20OR%20res_format%3AWMS)",
+        "fq=+(res_format%3Akmz%20OR%20res_format%3Ageojson%20OR%20res_format%3Acsv-geo-au%20OR%20res_format%3Aaus-geo-csv)"      
```

Comment:
Should have noticed this before I merged it: there's no `res_format` for KML listed, which means that you're probably only getting KMLs that are available in other formats, too.